lib/cdo/graphics/certificate_image.rb

```diff
@@ -131,27 +127,8 @@ def create_course_certificate_image(name, course=nil, sponsor=nil, course_title=
     course_title ||= fallback_course_title_for(course)
 
     image = Magick::Image.read(path).first
-
-    # student name
-    name_vertical_offset = 445
-    Magick::Draw.new.annotate(image, 0, 0, 0, name_vertical_offset, name) do
-      self.gravity = Magick::NorthGravity
-      self.pointsize = 96
-      self.font_family = 'Helvetica'
-      self.font_weight = Magick::BoldWeight
-      self.stroke = 'none'
-      self.fill = 'rgb(118,101,160)' # purple
-    end
-
-    course_vertical_offset = 610
-    Magick::Draw.new.annotate(image, 0, 0, 0, course_vertical_offset, course_title) do
-      self.gravity = Magick::NorthGravity
-      self.pointsize = 60
-      self.font_family = 'Helvetica'
-      self.font_weight = Magick::BoldWeight
-      self.stroke = 'none'
-      self.fill = 'rgb(29, 173, 186)' # teal
-    end
+    apply_text(image, name, 75, 'Helvetica bold', 'rgb(118,101,160)', 0, -141)
```

Comment:
just double-checking for my own edification: is there any difference between using the font "Helvetica bold" and using the font "Helvetica" with the "bold" weight?

Reply:
I vaguely remember trying to set a "bold" weight when originally doing the HOC certificates but it didn't seem to affect the text when it was rendered through Pango. I could try looking into it more though if you want.

Reply:
I guess the before/after screenshots are actually sufficient to convince me there is no difference